lib/teams_manager.rb

```diff
@@ -16,5 +16,28 @@ def generate_list(data_path)
     list_of_data
   end
 
+  def count_of_teams #ask intructors if they want this
+    teams.uniq.count
+  end
+
+  def get_names_hash
+    team_names_hash = {}
+    @teams.each do |team|
+      team_names_hash[team.team_id] = team.team_name
+    end
+    team_names_hash
+  end
+
+  def get_team_name(team_id)
+    name_hash = get_names_hash
+    name_hash[team_id]
+  #alternate replacement without hashing.
+  #get_name(id)
+    # team = @teams.find do |team|
+    #   team.team_id == id
+    # end
+    # team.name
+  #end
+  end
```

Comment:
what bozo wrote this code?

Reply:
serious note: we need to decide wether we want to keep the hash approach, or not.